MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -885,4 +979,17 @@
 
     <!-- This string is displayed in the content language list in front of the system default language in between parenthesis -->
     <string name="language_default">Default</string>
+
+    <!-- This string is displayed in the URL bar when the user is in the home page -->
```

Comment:
Please explain what "%1$s" refers to in comment.
CC @keianhzo.

Reply:
Fixed: https://github.com/MozillaReality/FirefoxReality/pull/1789